fractal_server/app/models/workflow.py

```diff
@@ -40,6 +40,7 @@ class Config:
     task_id: Optional[int] = Field(foreign_key="task.id")
 
     order: Optional[int]
+    executor: Optional[str] = None
```

Comment:
Ambiguity between property and attribute names.

Reply:
Attention!
you are adding to the base model an attribute that is specific to a single backend. Please consider subclassing within the backend.